cds_migrator_kit/transform/xml_processing/rules/base.py

```diff
@@ -112,6 +112,8 @@ def process_contributors(key, value):
     }
     if role:
         contributor.update({"role": {"id": role}})
+    elif key == "701__":
+        contributor.update({"role": {"id": "supervisor"}})
```

Comment:
is this assumption safe?

Reply:
yes, this assumption is confirmed with the library